feat: added missing statuses. (#2486)
Mergifyio/mergify-engine: docs/source/conditions.rst

       `continuous-integration/travis-ci/pr` or of a *check run* such as
       `Travis CI - Pull Request`. See `About Status Checks`_ for more
       details.

>        Also checks that report to be cancelled, timed out, and action
>        required are also considered as failures.

Reviewer: ```suggestion
       Checks that report being cancelled, timed out, and action
       required are also considered as failures.
```
